blocks/v2-columns/v2-columns.css

```diff
@@ -18,13 +22,25 @@
 }
```

Comment:
can you remove this max-width just for this variant to be applied from min-width: 744px. it looks odd from 560px to 743px IMHO
<img width="593" alt="Screenshot 2024-01-23 at 12 12 39" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/117984204/94eb89da-41ec-4461-a4d5-41234de67203">
<img width="760" alt="Screenshot 2024-01-23 at 12 12 59" src="https://github.com/Netcentric/vg-macktrucks-com-rd/assets/117984204/2e03f43c-16b0-4fdb-9f1e-f19c99686023">

Reply:
then I approve it in a moment